operations/compare-helm-with-jsonnet/components/config/irrelevant-config.yaml

```diff
@@ -81,3 +81,6 @@ config:
   common:
     storage:
       backend:
+  limits:
+    ruler_alertmanager_client_config:
+      alertmanager_url:
```

Comment:
This is not really new. It's already marked further up in the file as irrelevant for the helm/jsonnet differ: https://github.com/grafana/mimir/blob/9d620ef5cf9a2c595eb48b2a0ecc47033e1a3492/operations/compare-helm-with-jsonnet/components/config/irrelevant-config.yaml#L39

Since the config is in a new spot, we have to suppress it in the new spot too.